index.js

```diff
@@ -0,0 +1,7 @@
+import Store from './modules/books_functions.js';
+import configNavItems from './modules/nav_functions.js';
+import setdate from './modules/current_datetime.js';
+
+Store.init();
+configNavItems();
+setdate();
```

Comment:
- [OPTIONAL] Kindly make sure the app navigation is working, so the user can be able to add the new book
- [OPTIONAL] KIndly make sure the date and time are displayed in the page correctly.

   ### YOUR APP BEHAVIOUR

     ![chrome-capture-2023-5-6](https://github.com/AndreaM2429/Awesome_books/assets/87197412/62e2f063-818d-408c-a201-2e40b3deb8e3)